toc/rfc/rfc-draft-pcap-bosh.md

```diff
@@ -2,60 +2,214 @@
+| 3 | Concurrent capturing requests from different users or IP addresses | There is no enforced limit for concurrent captures. An accidental overload is possible more easily.                                                                                                                 |
+| 4 | Filter out traffic between capturing agent and target              | Traffic is filtered out by excluding the IP address(es) of the requester. The ports of the SSH connections can be included in the exclusion for more fine-grained control.                                          |
+| 5 | Output of captured data to a local file or stdout                  | The merged pcap data can be redirected to a file or configurable stream (e.g. stdout).                                                                                                                              |
+| 6 | Clean shutdown and error handling                                  | `tcpdump` [accepts signals to control ongoing captures](https://www.tcpdump.org/manpages/tcpdump.1.html). This includes clean shutdown with `SIGINT`/`SIGTERM` and flushing already captured buffers via `SIGUSR2`. |
+| 7 | Authorized Access                                                  | Access authorization can be enforced similar to BOSH SSH. The same SSH keys can be re-used as well.                                                                                                                 |
+
+All of those use cases would need to be re-assessed for Windows based stemcells.
+
+#### Summary
+
+The "pcap-lite" solution of more advanced orchestration of streamed `tcpdump` based captures via SSH provides a lean alternative to the pcap-release.
+
+Some features are missing or not a extensive as with the pcap-release integration. Most notably, congestion control is missing.
+
+For the initial problem described in this RFC, this is a "Minimal Viable Product" that provides the functionality for the support and debugging case and offers great functionality that covers a majority of use cases. 
+
+Further developments or more advanced use cases may be hindered by its simplicity however and will need to be re-assessed as they become relevant.
+
+### Comparison of Solutions
+
+This section provides a comparison of both presented solutions, highlighting their benefits and drawbacks. This comparion is particularly relevant for the community's decision for either or none of the presented solutions.
+
+#### 1. pcap-release integration into BOSH
+
+Main benefits of the pcap-release integration approach are:
+* Extended functionality through bidirectional signalling, including congestion control.
+* Clear signalling of state, errors during communication and status informaiton for the user.
+* Central component with enforceable limits, e.g. in terms of concurrent captures.
+* Portable Go code should support any stemcell (including Windows), for which libpcap is available.
```

Comment:
This is a bold claim.

Reply:
Reduced boldness. Changed this to be about being self-contained code without the need to call external processes.